src/services/course-service.ts

```diff
@@ -47,16 +34,14 @@ export const CourseService = {
       })
     })
   },
-  addCourse: async (data?: CourseForm): Promise<AxiosResponse> =>
-    await axiosClient.post(URLs.courses.create, data),
-  addCourseQuery: async (data: CourseForm) => {
+  addCourse: async (data: CourseForm) => {
```

Comment:
```suggestion
  addCourse: (data: CourseForm) => {
```

Why do you need ```async``` here?